### PR TITLE
Direction finding connectionless beacon sample app: add support for nRF52820

### PR DESCRIPTION
**Pull request by ppryga**
Add required configuration and DTS overlay for support the nRF52820 SOC.

This PR should be merged after: https://github.com/nrfconnect/sdk-nrf/pull/4773
Also upmerge of sdk-zephyr repository has to be finished.

### CODE REVIEW

File: samples/bluetooth/direction_finding_connectionless_tx/README.rst [hunk 1]
```diff
@@ -17,6 +17,7 @@ The sample supports the following development kits:
 .. table-from-rows:: /includes/sample_board_rows.txt
    :header: heading
    :rows: nrf52833dk_nrf52833
```
Comment (MarekPieta) [inline samples/bluetooth/direction_finding_connectionless_tx/README.rst:19]: `:rows: nrf52833dk_nrf52833, nrf52833dk_nrf52820`

File: samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52820.overlay [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
```
Comment (MarekPieta) [inline samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52820.overlay:4]: `SPDX-License-Identifier: LicenseRef-Nordic-5-Clause` ?

File: samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52820.overlay [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+&radio {
+	status = "okay";
+	/* This is an example number of antennas that may be available
+	 * on antenna matrix board.
+	 */
+	dfe-antenna-num = < 10 >;
```
Comment (MarekPieta) [inline samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52820.overlay:12]: We are not going to use configuration for antenna matrix designed by Nordic here?

**Review by MarekPieta — CHANGES_REQUESTED** [review]

File: samples/bluetooth/direction_finding_connectionless_tx/README.rst (hunk 1 above)
Comment (ppryga) [reply samples/bluetooth/direction_finding_connectionless_tx/README.rst]: Corrected

File: samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52820.overlay (hunk 3 above)
Comment (ppryga) [reply samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52820.overlay]: The content of the file was wrong. Updated to correct one.

File: samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52820.overlay (hunk 2 above)
Comment (ppryga) [reply samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52820.overlay]: The content of the file was wrong. Updated to correct one.

**Review by MarekPieta — APPROVED** [review]

**Review by KAGA164 — APPROVED** [review]

**Review by joerchan — APPROVED** [review]

**Comment by ppryga** [comment]
@mia-ko could you take a look on documentation here?

**Review by mia-ko — APPROVED** [review]

**Pull request merged**